cdap-ui/app/cdap/components/DatasetDetailedView/index.js

```diff
@@ -157,7 +157,7 @@ export default class DatasetDetailedView extends Component {
 
             let entityDetail = {
               programs,
-              schema: res[0].schema,
+              schema: res[0].properties[0].value,
```

Comment:
this is quite unsafe since there is no guarantee that the first element in the properties array will always be the schema. Instead I think because in line 152-156 we are already creating a properties object, perhaps get the schema from there instead?